ipl-auction-react/src/control/outLinedTextField.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./outLinedTextField.css";
+
+const OutLinedTextFiled = (props) => {
+  const {
+    id,
+    type,
+    value,
+    checked,
+    disabled,
+    placeholder,
+    onChange,
+    fieldName,
+  } = props;
+  return (
+    <div className="textfield">
+      <input
+        id={id}
```

Comment:
why we are getting id from parent component

Reply:
The id is a propTypes of string to get the special property to take the element, especially from the DOM to manipulate something.